App.js

```diff
@@ -1,28 +1,27 @@
 import 'react-native-gesture-handler';
-import 'react-native-gesture-handler';
 import * as React from 'react';
 // import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 import LoginView from './components/LoginView/LoginView';
 import Map from './components/Map/Map';
 import HeaderNav from './components/HeaderNav/HeaderNav';
 import ShortEventCard from './components/ShortEventCard/ShortEventCard';
 import FullEventCard from './components/FullEventCard/FullEventCard';
 import CreateEventForm from './components/CreateEventForm/CreateEventForm';
+import Dashboard from './containers/Dashboard/Dashboard'
```

Comment:
If we are adding the Dashboard component, can we get rid of the Map, HeaderNav, and ShortEventCard component imports right here?